templates/registration/login.html

```diff
@@ -16,25 +16,34 @@
   {% endif %}
 {% endif %}
 
-<form method="post" action="{% url 'login' %}">
+<form method="post" action="{% url 'login' %}" class="uk-form uk-form-horizontal maxw600">
```

Comment:
Should you also add  `form.non_field_errors`? For example if a user has been deactivated, or made too many login attempts, this would probably go to `non_field_errors`.